lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -37,7 +36,7 @@
   <li class="field">
     <label>${_("Message:")}</label>
       <div class="email-editor">
-      ${ HTML(section_data['editor']) }
+      ${section_data['editor']}
```

Comment:
@douglashall: We reverted back to the original recommendation, so ignore
this.  Thanks!

On Fri, Apr 8, 2016 at 3:33 PM, Douglas Hall <notifications@github.com>
wrote:

> In lms/templates/instructor/instructor_dashboard_2/send_email.html
> <https://github.com/edx/edx-platform/pull/12082#discussion_r59077568>:
>
> > @@ -37,7 +36,7 @@
> >    <li class="field">
> >      <label>${_("Message:")}</label>
> >        <div class="email-editor">
> > -      ${ HTML(section_data['editor']) }
> > +      ${section_data['editor']}
>
> @robrap <https://github.com/robrap>: Thanks for the heads up.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/12082/files/eefbb70439852207764f3c1bdafcb2ad57d0b14b#r59077568>
>



-- 

*Robert Raposa*

edX | Software Architect  | rraposa@edx.org

141 Portland Street, 9th floor

Cambridge, MA 02139
http://www.edx.org <http://www.edxonline.org/>

[image:
http://www.e-learn.nl/media/blogs/e-learn/edX_Logo_Col_RGB_FINAL.jpg?mtime=1336074566]